.travis.yml

```diff
@@ -0,0 +1,18 @@
+sudo: false
+language: go
+go:
+  - 1.6.3
```

Comment:
Go 1.6.3 is a bit old.
I think 1.6.4 or 1.7.5, 1.8 is better.
Please refer https://golang.org/doc/devel/release.html